sites/kit.svelte.dev/src/routes/home/Hero.svelte

```diff
@@ -0,0 +1,146 @@
+<script>
+	import logotype from './svelte-kit-logotype.svg';
+	import background from './svelte-kit-machine.webp?w=1440;800;600;480&format=avif;webp;png&picture';
+</script>
+
+<section class="hero">
+	<div class="hero-contents">
+		<div class="hero-text">
+			<img alt="SvelteKit logotype" width="400" height="50" class="logotype" src={logotype} />
+			<div class="tagline">web development, streamlined</div>
+			<a class="cta" href="/docs/introduction">read the docs</a>
```

Comment:
Axe flagged color contrast issues on this CTA.

> Element has insufficient color contrast of 3.5 (foreground color: #ffffff, background color: #ff4000, font size: 15.0pt (20px), font weight: normal). Expected contrast ratio of 4.5:1

Reply:
How serious do we think this is? It kinda feels like one of those cases where automated checking just doesn't behave the way human eyes do. If I reduce the lightness far enough that we get 4.5:1, it's a muddy mess